### PR TITLE
fix initialization of $post_int in EF_Module->get_current_post_type()

### PR DESCRIPTION
**Pull request by jeremyclarke**
It was a line that just said "$post_int;" which does nothing AFAIK.

In Debug Bar I was seeing this error:
 NOTICE: /class-module.php:243 - Undefined variable: post_int

The whole method could use a review but this PR is the minimum possible change to fix the notice and (I think) honor the original intention of the code. 

Thanks!

### CODE REVIEW

File: common/php/class-module.php [hunk 1]
```diff
@@ -253,7 +253,7 @@ function enqueue_datepicker_resources() {
 	function get_current_post_type() {
 		global $post, $typenow, $pagenow, $current_screen;
 		//get_post() needs a variable
-		$post_int;
+		$post_int = '';
```
Comment (mjangda) [inline common/php/class-module.php]: Can probably simplify to this:

```
$post_int = isset( $_REQUEST['post'] ) ? (int)$_REQUEST['post'] : false;
```

In any case, we need to change one of the `elseif` blocks below:

```
&& isset( $_REQUEST['post'] )
 			&& isset( $post_int )
```

to

```
&& $post_int
```

Comment (jeremyclarke) [inline common/php/class-module.php]: Sure! Figured I'd let you fix it up if you wanted. As long as the notice goes away I'm happy :)

**Comment by cojennin** [comment]
@jeremyclarke do you have some time to make the finishing touches on this PR per @mjangda's comments? Should be good to merge after they're made.

**Comment by cojennin** [comment]
Closing per #345

**Pull request closed without merging**